functionary/ui/templates/partials/table.html

```diff
@@ -1,2 +1,25 @@
 {% load django_tables2 %}
-{% render_table table 'django_tables2/bootstrap5.html' %}
+{% load widget_tweaks %}
+{% if filter %}
+    <form action="" method="get" class="form row">
+        {% for field in filter.form %}
+            <div class="col-auto mb-3 ">
+                {{ field|add_label_class:"form-label" }}
```

Comment:
Small thing, but can we get rid of the ":" in the label?  Right now it's something like "Function:", but the ":" is kind of superfluous.